geomstats/vectorization.py

```diff
@@ -14,32 +14,85 @@
 ERROR_MSG = "Invalid type: %s."
 
 
-def get_n_points(points, point_type):
+def _get_max_ndim_point(*point):
```

Comment:
What is the philosophy behind the use of `get`? Why not `max_ndim_point` which is shorter + more consistent with the current coding style + more consistent with pythonic style generally speaking?

https://stackoverflow.com/questions/374763/should-i-use-get-set-prefixes-in-python-method-names

Reply:
see comment above. related with this idea that when I call the function I'll likely want to use the same name. having a verb helps avoiding the conflict.